### PR TITLE
Added header and footer

### PR DESCRIPTION
**Pull request by RaulTsc**

### CODE REVIEW

File: src/universal/common/components/AppHeader/AppHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {Link} from 'react-router';
+import {AppLogo} from 'universal/common/components';
+
+export default class AppHeader extends React.Component<any, any> {
+  state = {
+    isScrolled: false
```
Comment (liviuignat) [inline src/universal/common/components/AppHeader/AppHeader.tsx]: We don't need the `isScrolled` logic here. This was supposed to set a state variable when the window was scrolled, to be able to make the header smaller when scrolling.

File: src/universal/common/components/AppHeader/AppHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  };
+
+  componentDidMount() {
```
Comment (liviuignat) [inline src/universal/common/components/AppHeader/AppHeader.tsx]: Part of the `isScrolled` logic. `componentDidMount` should be removed.

File: src/universal/common/components/AppHeader/AppHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    if (window && window.addEventListener) {
+      window.addEventListener('scroll', this.handleScroll.bind(this));
+    }
+  }
+
+  componentWillUnmount() {
```
Comment (liviuignat) [inline src/universal/common/components/AppHeader/AppHeader.tsx]: Part of the `isScrolled` logic. Should be removed.

File: src/universal/common/components/AppHeader/AppHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    if (window && window.removeEventListener) {
+      window.removeEventListener('scroll', this.handleScroll.bind(this));
+    }
+  }
+
+  handleScroll(ev) {
+    const {scrollTop} = ev.srcElement.body;
+    const isScrolled = !!scrollTop;
```
Comment (liviuignat) [inline src/universal/common/components/AppHeader/AppHeader.tsx]: Part of the `isScrolled` logic. Should be removed.

File: src/universal/common/components/AppHeader/AppHeader.scss [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+  @media #{$screen-xs} {
+    padding: 0 36px;
+  }
+
+  * {
+    transition: all 0.3s ease;
+  }
+
+  &[data-is-scrolled="true"] {
```
Comment (liviuignat) [inline src/universal/common/components/AppHeader/AppHeader.scss]: Part of the `isScrolled` logic. Should be removed.

File: src/universal/common/components/AppHeader/AppHeader.scss [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+  transition: all 0.3s ease;
+  padding: 0 8px;
+  z-index: 1;
+  box-shadow: 0 2px 2px 1px rgba(0, 0, 0, 0.1);
+
+  @media #{$screen-xs} {
+    padding: 0 36px;
+  }
+
+  * {
```
Comment (liviuignat) [inline src/universal/common/components/AppHeader/AppHeader.scss]: Also the transition should be removed, there is no more animation when scrolling :).

**Comment by RaulTsc** [comment]
I intentionally left that logic there thinking we might need it later. We don't need it for login/register/resetpassword but we might need it later in the app?

**Comment by liviuignat** [comment]
Nope, we just need this for the `er-homepages`.

**Comment by RaulTsc** [comment]
Should I rebase & force push? @liviuignat

**Comment by liviuignat** [comment]
@RaulTsc yep, you could rebase on `develop`

**Pull request merged**